stdlib/public/Concurrency/Actor.cpp

```diff
@@ -50,6 +50,9 @@
 
 #if HAVE_PTHREAD_H
 #include <pthread.h>
+#if HAVE_PTHREAD_NP_H
```

Comment:
Unfortunately, Swift doesn't have it's own fully self-contained configuration.  HAVE_PTHREAD_H should be coming from the LLVM configuration.   Rather than add a `pthread_np.h` check there, I suggest you stick with the `__has_include` for this file for now until we can rework Swift's CMake infrastructure.